opensrp-app/src/main/java/org/smartregister/commonregistry/CommonRepository.java

```diff
@@ -157,12 +184,24 @@ public void mergeDetails(String caseId, Map<String, String> details) {
         database.update(TABLE_NAME, valuesToUpdate, ID_COLUMN + " = ?", new String[]{caseId});
     }
 
+    public String[] getTableColumns() {
+        String[] columns = new String[common_TABLE_COLUMNS.length];
```

Comment:
We can populate this only once since it's called multiple times and doesn't change after initialization of columns

Reply:
Done